script/config/podman-compose.yml

```diff
@@ -3,88 +3,134 @@ services:
   redis-master:
     image: redis:6.2-alpine
     container_name: redis-master
-    network_mode: "host"
-    ports:
-      - "6379:6379"
+    network_mode: host
```

Comment:
btw, host mode is the default for root. If we just want to connect the different things together, then also running containers inside a single pod is a good way to isolate. But just saying, it's fine to stay.

Reply:
Is there a particular way to run the service inside a pod? I though they were already doing so. Also, I need some of them, like the master or the sentinels to be accessible from the test suite which runs in the Ubuntu VM in CircleCI. I tried a lot of things and only `network_mode: host` worked.

Reply:
For accessing containers from host, one would need to map ports when starting containers.

Reply:
I tried port mapping first but if I remember correctly I couldn't manage to make containers see each other. Maybe sharing pod could have helped.